cmake/linux/package_linux.sh.in

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+# Creates Linux ".AppImage" for @PROJECT_NAME_UCASE@
```

Comment:
> Why are you going through all the hassle of making this part of CMake when a few lines of .travis.yml or shell script would do?

I hope I explained this (same?) question properly in https://github.com/LMMS/lmms/pull/3688#issuecomment-313826333.

> Do you think it would be possible/advantageous to have a generic CMake AppImage module/plugin? If so, would you know where to start?

Hesitant "yes" to "advantageous ... cmake module" question.

A "no" to knowing where to start.  The first that comes to mind is CMake's NSIS integration, but it's [really ugly](https://github.com/LMMS/lmms/blob/stable-1.2/cmake/nsis/CMakeLists.txt#L16) to use.

CMake also offers [`CPackDMG`](https://cmake.org/cmake/help/v3.6/module/CPackDMG.html) for Mac, but it wasn't really useful for us.  Our project requires `macdeployqt` combined with manual relinking.   So from my perspective it's much easier to test, troubleshoot and patch in a common scripting language (which is `bash` for the time being)